server/doi/api.js

```diff
@@ -8,7 +8,9 @@ import { getPermissions } from './permissions';
 const previewOrDepositDoi = async (req, options = { deposit: false }) => {
 	const { deposit } = options;
 	const user = req.user || {};
-	const { target, communityId, collectionId, pubId } = deposit ? req.body : req.query;
+	const { target, communityId, collectionId, pubId, contentVersion } = deposit
```

Comment:
Looking at this with fresh eyes, I think it's better to have the callers pass in `req.body` or `req.query` as appropriate, rather than using the implicit assumption that `deposit: false` means that `req.method = 'GET'`.